osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/resources/jasper-interface.properties

```diff
@@ -9,5 +9,7 @@ jwcc.username=TestUser
 jwcc.password=1234
 jwcc.getsession.retries=30
 jwcc.getsession.sleep.between.retries=10000
+# validityPeriod (0, 143) -> validityPeriod x 5 minutes = 5, 10, 15 minutes
```

Comment:
Default value is 0. 0 x 5 minutes = 0 minutes? Is this comment correct?

Reply:
- 
![cb9d8860-1fb4-4fe4-8b38-8c69c8bf90b8](https://user-images.githubusercontent.com/11977679/129175836-95490430-ea55-4ad7-b9ee-b5d913d8463a.png)